lib/rutema/core/engine.rb

```diff
@@ -128,17 +192,41 @@ def run_test specification
       end
       return status
     end
+
+    ##
+    # Instantiate a new class of a given type passing it a given configuration
+    # upon construction
+    #
+    # * +definition+ - class of which a new instance shall be instantiated
+    # * +configuration+ - Configuration instance which shall be passed to the
+    #   initializer of the to be created instance
     def instantiate_class definition,configuration
       if definition[:class]
         klass=definition[:class]
         return klass.new(configuration)
       end
       return nil
     end
+
+    ##
+    # Check if the given test identifier belongs to the normal test cases or to
+    # one of the special ones (the test (suite) setups and teardowns)
+    #
+    # * +test_identifier+ - the test identifier which shall be checked if it
```

Comment:
the test identifier to check against membership in the normal or special test case identifier sets.

Reply:
I went with your wording :+1: